wlauto/config_example.py

```diff
@@ -286,4 +286,8 @@
 # Scratch resource getter.
 #scratch_mount_point = '/arm/scratch'
 
+# URL of server from where to fetch external resources for workloads on-demand.
+# Used by WA's get-assets sub-command.
+remote_assets_url = 'https://github.com/jimboatarm/wa-assets/raw/master/dependencies'
```

Comment:
So the order of priority would be:
1. `--url` argument if specified
2. `remote_assets_url` config if present
3. default value inside `get-assets` script

Does that look OK?